### PR TITLE
Make mandatory elements mandatory

### LINKED ISSUE #7: Some elements refered to with minOccurs="0"
The following elements are refered to with `minOccurs="0"` in the XSD, though they have a cardinality of (at least) 1 according to Part 1:

- [`tt:styling`](https://github.com/ebu/ebu-tt-xsd/blob/214fdc3a6a2a0c91e22cd69a8d9b69a031b68b2a/ebutt.xsd#L39)
- [`tt:layout`](https://github.com/ebu/ebu-tt-xsd/blob/214fdc3a6a2a0c91e22cd69a8d9b69a031b68b2a/ebutt.xsd#L40)
- [`tt:region`](https://github.com/ebu/ebu-tt-xsd/blob/214fdc3a6a2a0c91e22cd69a8d9b69a031b68b2a/ebutt.xsd#L102)
- [`tt:div`](https://github.com/ebu/ebu-tt-xsd/blob/214fdc3a6a2a0c91e22cd69a8d9b69a031b68b2a/ebutt.xsd#L151)

### PR DESCRIPTION
**Pull request by nigelmegitt**
Make the following mandatory elements' cardinality:
* `tt:styling`
* `tt:layout`
* `tt:region`
* `tt:div`

Also describe a style guide in the README where if non-default `minOccurs` or `maxOccurs` values are used, _both_ attributes should be specified.

Closes #7.

### CODE REVIEW

**Comment by nigelmegitt** [comment]
This may also close #9.

**Comment by nigelmegitt** [comment]
Now also closes #12.

File: README.md [hunk 1]
```diff
@@ -22,3 +22,11 @@ This schema depends on the w3c/ebu-tt-m-xsd repository and imports it via a git
 
 If cloning from the command line make sure to include the `--recurse-submodules` option,
 or after cloning run `git submodule init` then `git submodule update`.
+
+## Style guide
+
+If either the `minOccurs` or `maxOccurs` attributes on an element do not have their default value of 1,
+_both_ attribtues should be specified.
```
Comment (spoeschel) [inline README.md]:
```suggestion
_both_ attributes should be specified.
```

File: ebutt.xsd [hunk 2]
```diff
@@ -98,8 +98,8 @@ Please note that the EBU-TT XML Schema is a helping document and NOT normative b
 			<xs:documentation>Container element for layout information.</xs:documentation>
 		</xs:annotation>
 		<xs:sequence>
-		    <xs:element name="metadata" type="ebuttm:anyMetadata_type" minOccurs="0"/>
-			<xs:element name="region" type="tt:region" minOccurs="0" maxOccurs="unbounded"/>
+		    <xs:element name="metadata" type="ebuttm:anyMetadata_type" minOccurs="0" maxOccurs="1"/>
+			<xs:element name="region" type="tt:region"  minOccurs="1" maxOccurs="unbounded"/>
```
Comment (spoeschel) [inline ebutt.xsd]:
```suggestion
			<xs:element name="region" type="tt:region" minOccurs="1" maxOccurs="unbounded"/>
```

File: ebutt.xsd [hunk 3]
```diff
@@ -29,23 +29,23 @@ Please note that the EBU-TT XML Schema is a helping document and NOT normative b
...
 		</xs:sequence>
 	</xs:complexType>
 	<xs:complexType name="styling">
 		<xs:annotation>
 			<xs:documentation>Container for styling information.</xs:documentation>
 		</xs:annotation>
 		<xs:sequence>
-		    <xs:element name="metadata" type="ebuttm:stylingMetadata_type" minOccurs="0"/>
+		    <xs:element name="metadata" type="ebuttm:stylingMetadata_type" minOccurs="0" maxOccurs="1"/>
 			<xs:element name="style" type="tt:style" maxOccurs="unbounded"/>
```
Comment (spoeschel) [inline ebutt.xsd]:
```suggestion
			<xs:element name="style" type="tt:style" minOccurs="1" maxOccurs="unbounded"/>
```

**Review by spoeschel — CHANGES_REQUESTED** [review]
Thank you! I just suggested some small fixes.

**Comment by nigelmegitt** [comment]
Thank you, all those changes accepted.

**Review by spoeschel — APPROVED** [review]

**Pull request merged**